ToggleAppliances/Main.cs

```diff
@@ -1,24 +1,28 @@
-﻿using Harmony;
-using System;
-using System.IO;
-using System.Reflection;
-
-namespace ToggleMachines
-{
-    public class Main
-    {
-        public static void Patch()
-        {
-            var harmony = HarmonyInstance.Create("com.ahk1221.toggleappliances");
-            harmony.PatchAll(Assembly.GetExecutingAssembly());
-
-            Logger.Log("Successfully patched!");
-        }
-
-        public static string GetSavePathDir()
-        {
-            var savePathDir = Path.Combine(@".\SNAppData\SavedGames\", Utils.GetSavegameDir());
-            return Path.Combine(savePathDir, "ToggleAppliances");
-        }
-    }
-}
+﻿using Harmony;
+using System;
+using System.IO;
+using System.Reflection;
+using UnityEngine;
+using QModManager.API.ModLoading;
+
+namespace ToggleMachines
+{
+    [QModCore]
+    public class Qpatch
+    {
+        [QModPatch]
+        public static void Patch()
+        {
+            var harmony = HarmonyInstance.Create("com.ahk1221.toggleappliances");
+            harmony.PatchAll(Assembly.GetExecutingAssembly());
+
+            Logger.Log("Successfully patched!");
+        }
+
+        public static string GetSavePathDir()
+        {
+            var savePathDir = Path.Combine(@".\SNAppData\SavedGames\", SaveLoadManager.GetTemporarySavePath());

```

Comment:
Just noticed this issue:  You don't need to include anything else to the start of the path from `SaveLoadManager.GetTemporarySavePath()` to get the proper save game path.

You can simplify this method to combine only the value from `GetTemporarySavePath` and the "ToggleAppliances" folder.

Examples:  
- [AuxCyUpgradeConsoleSaveData.cs#L68](https://github.com/PrimeSonic/PrimeSonicSubnauticaMods/blob/master/MoreCyclopsUpgrades/AuxConsole/AuxCyUpgradeConsoleSaveData.cs#L68)
- [CyBioReactorSaveData.cs#L64](https://github.com/PrimeSonic/PrimeSonicSubnauticaMods/blob/master/BetterBioReactor/SaveData/CyBioReactorSaveData.cs#L64)
- [CyNukeReactorSaveData.cs#L11](https://github.com/PrimeSonic/PrimeSonicSubnauticaMods/blob/master/CyclopsNuclearReactor/CyNukeReactorSaveData.cs#L11)